File: Algorithm/Programmers/Python/Level 1/문자열 내 p와 y의 개수.py
from collections import Counter


def solution(s):
    c = Counter(s)
    p = c.get("p", 0) + c.get("P", 0)
    y = c.get("y", 0) + c.get("Y", 0)

    if p == y:
        return True
    else:
        return False

File: Algorithm/Programmers/Java/Level 1/문자열_내_p와_y의_개수.java
class Solution {
    boolean solut문자열_내_p와_y의_개수ion(String s) {
        s = s.toUpperCase();
        int p = counter(s, "P");
        int y = counter(s, "Y");
        if (p == y) {
            return true;
        }
        return false;
    }

    int counter(String s, String c) {
        return s.length() - s.replace(c, "").length();
    }
}